scripts/compile/compile-stage.ps1

```diff
@@ -79,6 +79,7 @@ cp -rec "$RuntimeOutputDir\*" "$OutputDir\bin"
 
 # Deploy the CLR host to the output
 cp "$HostDir\corehost.exe" "$OutputDir\bin"
+cp "$HostDir\clihost.dll" "$OutputDir\bin"
```

Comment:
The name `clihost` seems a little odd to me. Is it referring to the dotnet-cli? In which case it is inaccurate since it is only **used** by the CLI, it is not connected directly to the cli. Is it referring to the Common Language Infrastructure? In which case it seems to close to `clrhost` which was discarded as the original name for this host because such a component already existed.

Can we just call it `corehost.dll`? The main idea is that it's just `corehost` in DLL form, right? I don't think there will be a big problem with having a `corehost.exe` and a `corehost.dll`